Stripe/StripeiOSTests/STPAddressTests.swift

```diff
@@ -140,334 +140,6 @@ class STPAddressTests: XCTestCase {
         XCTAssertNil(address.country)
     }
 
-    func testPKContactValue() {
```

Comment:
Can we keep these around? I think they're still used.

Reply:
Most of these tests use containsRequiredFields or containsContent, which is being removed from [STPAddress+BasicUI.swift](https://github.com/stripe/stripe-ios/blob/c8f86df3/Stripe/StripeiOS/Source/STPAddress+BasicUI.swift#L43). I can restore testPKContactValue, I think that one's still fine

Reply:
Ah, got it! Nw then, I think we can delete the ones that use containsRequiredFields or containsContent.